tests/smoke-tests/test-staking-rewards.ts

```diff
@@ -120,7 +121,16 @@ async function assertRewardsAt(api: ApiPromise, nowBlockNumber: number) {
         id: id,
         amount: amount,
       };
+      countedDelegationSum = countedDelegationSum.add(amount);
     }
+    const totalCountedLessTotalCounted = total.sub(countedDelegationSum.add(bond));
+    // expect(total.toString()).to.equal(
+    //   countedDelegationSum.add(bond).toString(),
+    //   `Total counted (denominator) ${total} - total counted (numerator
+    // ${countedDelegationSum.add(new BN(bond))} = ${totalCountedLessTotalCounted}` +
+    //     ` so this collator and its delegations receive fewer rewards for round ` +
+    //     `${originalRoundNumber.toString()}`
+    // );
```

Comment:
I'm leaving this commented out so that the inflation expect included in this PR provides output instead of erring early here.

I'm leaving it here because it will be useful for debugging when/if the inflation expect included later fails.